src/Engines/MeiliSearchEngine.php

```diff
@@ -145,6 +145,8 @@ protected function performSearch(Builder $builder, array $searchParams = [])
             unset($searchParams['filters']);
         }
 
+        $searchParams = array_merge($builder->options, $searchParams);
```

Comment:
I think it would be preferable to us, that users can not override the search params that we send to the engine.

if we do it like this, then our parameters for `limit/offset` (later `page/hitsPerPage` would still get to the underlyling engine.

(this suggestion also relates to the algolia engine)

```suggestion
        $searchParams = array_merge($searchParams, $builder->options);
```

Reply:
Hey @mmachatschek, I thought of that and that's what `array_merge` does: https://3v4l.org/a1JBo

Reply:
sorry about that @driesvints 👍 I mixed the contents of `$searchParams` and `$builder->options`